src/PrestaShopBundle/Form/Admin/Type/DeltaQuantityType.php

```diff
@@ -73,8 +89,7 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'delta_label' => $this->trans('Add or subtract items', 'Admin.Global'),
             ])
-            ->setAllowedTypes('delta_label', ['string', 'boolean', 'null'])
-        ;
+            ->setAllowedTypes('delta_label', ['string', 'boolean', 'null']);
```

Comment:
Just a suggestion, but we usually write with this kind of code convention:

```suggestion
            ->setAllowedTypes('delta_label', ['string', 'boolean', 'null'])
        ;
```